src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/questionWithResponses.tag

```diff
@@ -70,14 +71,15 @@
       <c:if test="${questionWithResponses.question.giverTeam}">
         <p class="text-warning">Please note that you are submitting this response on behalf of your team.</p>
       </c:if>
-
       <c:if test="${questionWithResponses.numOfResponseBoxes eq 0}">
         <p class="text-warning">${questionWithResponses.question.messageToDisplayIfNoRecipientAvailable}</p>
       </c:if>
 
       <c:forEach items="${questionWithResponses.responses}" var="response">
         <feedbackSubmissionEdit:response response="${response}" isSessionOpenForSubmission="${isSessionOpenForSubmission}"
-            questionWithResponses="${questionWithResponses}"/>
+            questionWithResponses="${questionWithResponses}" responseRecipientIndex="${response.responseIndx}"
+            responseGiverIndex="1" questionIndex="${questionWithResponses.question.qnIndx}"
```

Comment:
For any '1' in the template, you can use INDEX_FOR_FEEDBACK_SUBMISSION_PAGE_COMMENTS?